spec/models/registration_spec.rb

```diff
@@ -22,4 +22,28 @@
       end
     end
   end
+
+  describe '#destroy_purchased_tickets' do
+    it 'destroys purchased tickets if tickets are purchased' do
+      user = build(:user)
```

Comment:
You don't need the user, ``create(:registration)`` will [automatically](https://github.com/openSUSE/osem/blob/master/spec/factories/registration.rb#L5) create a user. So please remove this line!